downstream/modules/automation-hub/proc-install-ha-hub-selinux.adoc

```diff
@@ -0,0 +1,107 @@
+
+To set up a high availability (HA) deployment of {HubName} on SELinux, create two mount points for `/var/lib/pulp` and `/var/lib/pulp/pulpcore_static`, then assign the appropriate SELinux contexts to each. The current {PlatformNameShort} installer version requires you to first add the context for `/var/lib/pulp` and running the {PlatformNameShort} installer before adding the context for `/var/lib/pulp/pulpcore_static`.
+
+.Prerequisites
+* You have already configured a NFS export on your server.
+
+.Pre-installation procedure
+. Create a mount point at `/var/lib/pulp`:
++
+----
+$ mkdir /var/lib/pulp/
+----
+. Open `/etc/fstab` using a text editor, then add the following values:
++
+----
+srv_rhel8:/data /var/lib/pulp nfs defaults,_netdev,nosharecache 0 0
+srv_rhel8:/data/pulpcore_static /var/lib/pulp/pulpcore_static nfs defaults,_netdev,nosharecache,context="system_u:object_r:httpd_sys_content_rw_t:s0" 0 0
+----
+. Run the mount command for `/var/lib/pulp`:
++
+----
+$ mount /var/lib/pulp
+----
+. Create a mount point at `/var/lib/pulp/pulpcore_static`:
++
+----
+$ mkdir /var/lib/pulp/pulpcore_static
+----
+. Run the mount command:
++
+----
+$ mount -a
+----
+. With the mount points set up, run the {PlatformNameShort} installer:
++
+----
+$ setup.sh -- -b --become-user root
+----
+
+Once the installation is complete, unmount the `/var/lib/pulp/` mount point then apply the appropriate SELinux context:
+
+.Post-installation procedure
+. Shut down the Pulp service:
++
+----
+$ systemctl stop pulpcore.service
+----
+. Unmount `/var/lib/pulp/pulpcore_static`:
++
+----
+$ umount /var/lib/pulp/pulpcore_static
+----
+. Unmount `/var/lib/pulp/`:
++
+----
+$ umount /var/lib/pulp/
+----
+. Open `/etc/fstab` using a text editor, then replace the existing value for `/var/lib/pulp` with the following:
++
+----
+srv_rhel8:/data /var/lib/pulp nfs defaults,_netdev,nosharecache,context="system_u:object_r:pulpcore_var_lib_t:s0" 0 0
+----
+. Run the mount command:
++
+----
+$ mount -a
+----
+
+.Configure pulpcore.service:
+. With the two mount points set up, shut down the Pulp service to configure `pulpcore.service`:
++
+----
+$ systemctl stop pulpcore.service
+----
+. Edit `pulpcore.service` using `systemctl`:
++
+----
+$ systemctl edit pulpcore.service
+----
+. Add the following entry to `pulpcore.service` to ensure that {HubName} services starts only after starting the network and mounting the remote mount points:
++
+----
+[Unit]
+After=network.target var-lib-pulp.mount
+----
+. Enable `remote-fs.target`:
++
+----
+$ systemctl enable remote-fs.target
+----
+. Reboot the system:
++
+----
+$ reboot
+----
+
+.Troubleshooting
+A bug in the pulpcore SELinux policies can cause the token authentication public/private keys in `etc/pulp/certs/` to not have the proper SELinux labels, causing the pulp process to fail. When this occurs, run the following command to temporarily attach the proper labels:
```

Comment:
It would be nice to have an example of a typical error message, if it's available.